Engines/Wine/Settings/always offscreen/script.js

```diff
@@ -4,45 +4,50 @@ include("engines.wine.plugins.regedit");
 /**
  * setting to set always offscreen
 */
-var settingImplementation = {
-    _options: [tr("Default"), tr("Disabled"), tr("Enabled")],
-    // values which are written into the registry, do not translate!
-    _registryValues: ["", "disabled", "enabled"],
-    getText: function () {
+class AlwaysOffscreenSetting {
+    constructor() {
+        this.options = [tr("Default"), tr("Disabled"), tr("Enabled")];
+        // values which are written into the registry, do not translate!
+        this.registryValues = ["", "disabled", "enabled"];
+    }
+
+    getText() {
         return tr("Always offscreen");
-    },
-    getOptions: function () {
-        return this._options;
-    },
-    getCurrentOption: function (container) {
-        var currentValue = new Wine()
+    }
+
+    getOptions() {
+        return this.options;
+    }
+
+    getCurrentOption(container) {
```

Comment:
@plata do you think it makes sense to move the `container` argument to the constructor?
This would allow us to change this method to a normal JavaScript getter method:

```javascript
get currentOption() {
   ...
}
```

Reply:
I'm currently not quite sure where the objects are created. If they are created every time you select a container, we can do this. If they are created once and only called for different containers, it will not work.